src/extension.ts

```diff
@@ -604,6 +604,14 @@ export async function activate(context: vscode.ExtensionContext) {
 		]);
 	};
 
+	explorerTreeView.onDidChangeVisibility(({ visible }) => {
```

Comment:
can you move it closer to the `.reveal()` instruction in this file?

Reply:
addressed, merging PR